src/sentry/integrations/github/client.py

```diff
@@ -70,10 +70,18 @@ def get_repo(self, repo: str) -> JSONData:
         repository: JSONData = self.get(f"/repos/{repo}")
         return repository
 
-    # https://docs.github.com/en/rest/git/trees#get-a-tree
-    def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
-        tree = []
+    def get_repo_files(
```

Comment:
I'm changing this function a bit to only focus on return files for a repo. There's an option to return all files or just source code files.